src/client/automation/playback/click/select-child.js

```diff
@@ -45,7 +45,10 @@ export default class SelectChildClickAutomation {
             this.childIndex = isOption ? domUtils.getOptionIndex(this.parentSelect, this.element) :
                               domUtils.getOptionGroupIndex(this.parentSelect, this.element);
 
-            this.clickCausesChange = isOption && !this.element.disabled && this.childIndex !== selectedIndex;
+            var parentOptGroup = domUtils.isOptionGroupElement(this.element.parentNode) ? this.element.parentNode : null;
```

Comment:
No, it's not possible, browser inserts closing `</optgroup>` tag automatically before the nesetd optgroup.